apps.json

```
@@ -2739,6 +2739,12 @@
         "state": "working",
         "url": "https://code.antopie.org/miraty/qr_ynh"
     },
+    "question2answer": {
+        "branch": "master",
+        "revision": "HEAD",
+        "state": "working",
+        "url": "https://github.com/nils-van-zuijlen/question2answer_ynh"
+    },
```

```suggestion
    "question2answer": {
        "branch": "master",
        "category": "communication",
        "revision": "HEAD",
        "state": "working",
        "url": "https://github.com/nils-van-zuijlen/question2answer_ynh"
    },
```

(Not sure if that's the proper category, but proposing to put that in "communication" because Q2A is ~similar to a forum ?)

---

Yes, I agree with you, and it's a tool for **communicating** questions and getting answers